genConsole.ts

```diff
@@ -0,0 +1,173 @@
+import { writeFileSync } from "fs";
```

Comment:
Also move this to a different folder and add a generate command so we can regenerate this if some changes are needed